src/main/scala/MeanTimeBetweenResponse.scala

```diff
@@ -0,0 +1,31 @@
+import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.spark.SparkConf
+import org.joda.time.format.PeriodFormatterBuilder
+import org.joda.time.{DateTime, Period}
+
+//
+///**
+//  * Created by maxime on 23/11/15.
+//  */
+//
+object MeanTimeBetweenResponse{
+//
+  def getTweets(conf : SparkConf, inReplyToStatusId : Long, timestamp : DateTime ): String = {
+
+      if(inReplyToStatusId > 0){
+        CassandraConnector(conf).withSessionDo { session =>
+          val reply_id = session.execute("SELECT created_at FROM twitter_streaming.tweets WHERE tweet_id="+ inReplyToStatusId +" ALLOW FILTERING")
+          try {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/b-fovet/spark-streaming-twitter/file/1529237931/issues/source?bid=2980032&fileBranchId=3010315#l18)